sdk/storage/azure-storage-queue/azure/storage/queue/_serialize.py

```diff
@@ -19,8 +19,7 @@
 ]
 
 
-def get_api_version(kwargs):
-    # type: (Dict[str, Any]) -> str
+def get_api_version(kwargs: Dict[str, Any]) -> str:
```

Comment:
Lets revert this -- we want to make this in a big change for all packages

Reply:
Lets make this (`api_version: str`), make appropriate changes to clients (get it out of the kwargs to the *, ----)